src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -558,6 +558,28 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 {
                     assert(instrIsRMW);
 
+                    insScalableOpts sopt;
+
+                    switch (intrinEmbMask.id)
```

Comment:
I'm not sure if this is the best place to put this switch statement, but these intrinsics need to be wrapped with `ConditionalSelect`, so implementing special codegen for them off this path doesn't seem feasible without some more invasive refactoring.

Reply:
Yes, we do similar handling for `ConvertTo` for `numOperands == 1` case.